Automattic/wp-calypso
# Store: Move product stats to store

Per our chat earlier, this branch movies the MC stat bump and track events away from being done server side (`mu-plugins/tracks/hooks/woocommerce.php`) to being handled via Calypso. The events are renamed and I'll keep the PHP hooks running for a bit so we can compare numbers (since it's our assumption the product count is inflated).

To Test:
* Run `npm run test-client client/extensions/woocommerce` and make sure all tests pass.
* Open your console and paste `localStorage.setItem('debug', '*analytics*');`
* Create a product and make sure you see both the analytic event and the 'bump stat' event. 
* Update a product and make sure you see both the analytic event and the 'bump stat' event.
* Reset your setup flags and go through the address setup step, looking for `calypso_woo_store_setup_country` in your console.
* Following the FG page (linked on our P2), remove the blog sticker to make sure the stat bump is not blocked and retest.
